content/en/tracing/troubleshooting/_index.md

```diff
@@ -2,444 +2,109 @@
-
-If neither of these log entries is present, then no request was sent to the Agent, which means that the tracer is not instrumenting your application. In this case, [contact Datadog support][3] and provide the relevant log entries with [a flare][4].
-
-For more tracer settings, check out the [API documentation][5].
-
-[1]: https://datadog.github.io/dd-trace-js/Tracer.html#init
-[2]: /agent/troubleshooting/
-[3]: /help/
-[4]: /agent/troubleshooting/#send-a-flare
-[5]: https://datadog.github.io/dd-trace-js/#tracer-settings
-{{% /tab %}}
-{{% tab ".NET" %}}
-
-To enable debug mode for the Datadog .NET Tracer, set the `DD_TRACE_DEBUG` configuration setting to `true`. This setting can be set as an environment variable, in the `web.config` or `app.config` file (.NET Framework only), or in a `datadog.json` file. Debug mode can also be enabled in code by calling `GlobalSettings.SetDebugEnabled(true)`:
-
-```csharp
-using Datadog.Trace;
-
-// enable debug mode
-GlobalSettings.SetDebugEnabled(true);
-
-```
-
-Logs files are saved in the following directories by default. The `DD_TRACE_LOG_PATH` setting can be used to change these paths.
-
-| Platform | Path                                      |
-|----------|-------------------------------------------|
-| Windows  | `%ProgramData%\Datadog .NET Tracer\logs\` |
-| Linux    | `/var/log/datadog/dotnet/`                |
-
-**Note:**: On Linux, you must create the logs directory before you enabled debug mode.
-
-For more details on how to configure the .NET Tracer, see the [Configuration][1] section.
-
-
-[1]: /tracing/setup/dotnet/#configuration
-{{% /tab %}}
-{{% tab "PHP" %}}
-
-To enable debug mode for the Datadog PHP Tracer, set the environment variable `DD_TRACE_DEBUG=true`. See the PHP [configuration docs][1] for details about how and when this environment variable value should be set in order to be properly handled by the tracer.
-
-In order to tell PHP where it should put `error_log` messages, you can either set it at the server level, or as a PHP `ini` parameter, which is the standard way to configure PHP behavior.
-
-If you are using an Apache server, use the `ErrorLog` directive.
-If you are using an NGINX server, use the `error_log` directive.
-If you are configuring instead at the PHP level, use PHP's `error_log` ini parameter.
-
-[1]: https://www.php-fig.org/psr/psr-3
-{{% /tab %}}
-{{% tab "C++" %}}
-
-The release binary libraries are all compiled with debug symbols added to the optimized release. It is possible to use gdb or lldb to debug the library and to read core dumps. If you are building the library from source, pass the argument `-DCMAKE_BUILD_TYPE=RelWithDebInfo` to cmake to compile an optimized build with debug symbols.
-
-```bash
-cd .build
-cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo ..
-make
-make install
-```
-
-{{% /tab %}}
-{{< /tabs >}}
+If your tracer version includes these [startup logs][2], they are the recommended place to begin troubleshooting.
```

Comment:
None of the tracers will require any additional logging config similar to enabling debug mode correct?  This doc makes it sound like that initialize logging will happen by default  regardless of debug mode but maybe we should say so explicitly.